engines/xeen/spells.cpp

```diff
@@ -85,7 +85,8 @@ void Spells::executeSpell(MagicSpell spellId) {
 		&Spells::wizardEye
 	};
 
-	(this->*SPELL_LIST[spellId])();
+	if (spellId < 76)
+		(this->*SPELL_LIST[spellId])();
```

Comment:
I don't know if an error should be raised here if `spellId` is out of range

Reply:
Depends. If there's any place in the Russian version where a spell Id >= 76 is passed, it's better to have this as a workaround then to always crash the game